spec2vec/Spec2Vec.py

```diff
@@ -109,14 +117,14 @@ def matrix(self, references: List[SpectrumDocument], queries: List[SpectrumDocum
         """
         n_rows = len(references)
         reference_vectors = numpy.empty((n_rows, self.vector_size), dtype="float")
-        for index_reference, reference in enumerate(references):
+        for index_reference, reference in enumerate(tqdm(references, disable=self.disable_progress_bar)):
```

Comment:
Would be nice to see in stdout what the progress is for
```suggestion
        for index_reference, reference in enumerate(tqdm(references, desc='Calculating vectors of reference spectrums', disable=self.disable_progress_bar)):
```

Reply:
That looks like a nice solution, thanks.